receiver/prometheusreceiver/internal/otlp_metricfamily.go

```diff
@@ -170,10 +178,23 @@ func (mg *metricGroup) toDistributionPoint(orderedLabelKeys []string, dest *pmet
 	}
 	point.SetTimestamp(tsNanos)
 	populateAttributes(orderedLabelKeys, mg.ls, point.Attributes())
-
+	mg.setExemplars(exemplarsdata, point.Exemplars())
 	return true
 }
 
+func (mg *metricGroup) setExemplars(exemplarMap map[string][]exemplarData, exemplars pmetric.ExemplarSlice) {
+	metricKey := buildMetricKey(mg.ls)
+	ed := exemplarMap[metricKey]
```

Comment:
note that for histograms, this will only include exemplars on the _sum/_count series, since you are using `mg.ls` to generate the key.  This is because (as pointed out in otlp_transaction) the _bucket series of a histogram includes a unique `le` label to indicate which bucket it is.

Reply:
@dashpole Can you please help how this can be addressed?

Reply:
You need to remove the 'le' label from histograms, and the 'quantile' label from summaries, and then store the hash.  That would group all exemplars in the histogram/summary together into the same list.

Reply:
Got it, thanks

Reply:
@dashpole I tried building a label has without `le` and `quantile` but, exemplars were not generating so I tried printing `mg.ls` labels and they still have `le`. So the has wasn't matching.

Reply:
Do they all have the same value for `le`?

Reply:
@dashpole Yes, they all have same values for `le`